stream/src/main/java/org/apache/kafka/stream/processor/internals/IngestorImpl.java

```diff
@@ -15,7 +15,7 @@
  * limitations under the License.
  */
 
-package org.apache.kafka.clients.processor.internals;
+package org.apache.kafka.stream.processor.internals;
```

Comment:
@guozhangwang Why do we need to subscribe to all topics twice, once in the constructor and once in the open() method? Can this happen just once in the open() method?
https://github.com/confluentinc/kafka/pull/6/files#diff-ed20a63bab9ad85c9c581e9f187b3e37R49